static/app/utils/profiling/gl/utils.ts

```diff
@@ -624,3 +641,52 @@ export function computeHighlightedBounds(
 
   return null;
 }
+
+export function computeConfigViewWithStategy(
+  strategy: 'min' | 'exact',
+  view: Rect,
+  frame: Rect
+): Rect {
+  if (strategy === 'exact') {
+    return frame.withHeight(view.height);
+  }
+
+  if (strategy === 'min') {
+    if (view.width < frame.width) {
+      // If view width < frame width, we need to zoom out,
+      // so the condition = exact
+      return frame.withHeight(view.width);
```

Comment:
why is this setting the height to the width?

Reply:
It's a bug, I fixed it in the follow up PR, but forgot to fix it here too